server/routes/notification.js

```diff
@@ -0,0 +1,13 @@
+const express = require('express');
+const router = express.Router();
+const protect = require('../middleware/auth');
+const {
+  createNotification, readNotification, getAll, getUnread
+} = require('../controllers/notification');
+
+router.route('/create').post(protect, createNotification);
```

Comment:
```suggestion
router.route('/').post(protect, createNotification);
router.route('/:id').patch(protect, readNotification);
router.route('/').get(protect, getAll);
router.route('/unread').get(protect, getUnread);
```
We don't need to use verbs in the route name as the HTTP methods indicate the purpose of the route. We also want to use path params only when we want to identify a resource (it is not required when fetching notifications for the logged in user)

Reply:
Changed to suggested in commit.